### PR TITLE
[ready to merge] relabel MONDO:0013772 Huppke-Brendel syndrome

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
address #6765

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -274763,11 +274763,11 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0013772
-name: congenital cataract-hearing loss-severe developmental delay syndrome
+name: Huppke-Brendel syndrome
 subset: ordo_disease {source="Orphanet:300313"}
 synonym: "CCHLND" RELATED ABBREVIATION [MONDO:Lexical, OMIM:614482]
-synonym: "congenital cataract-deafness-severe developmental delay syndrome" EXACT [Orphanet:300313]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:274769]: why was this synonym referencing Orphanet removed?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -274763,11 +274763,11 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
...
-synonym: "congenital cataracts, hearing loss, and neurodegeneration" RELATED [MONDO:Lexical, OMIM:614482]
+synonym: "congenital cataracts, hearing loss, and neurodegeneration" EXACT [MONDO:Lexical, OMIM:614482]
+synonym: "congenital cataracts, hearing loss, and neurodegeneration" EXACT [OMIM:614482]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:274770]: What is the difference between `EXACT [MONDO:Lexical, OMIM:614482]` and `EXACT [OMIM:614482]`?

**Review by twhetzel — APPROVED** [review]
The label change looks fine. Two questions about the synonyms.

Also, since #6765 still has an open question, will that ticket remain open?

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:274770]: you are right, these should be combined into one line

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:274769]: It looks like a mistake. I think I may have meant to combine the two syns below and accidentally deleted this instead?

I'll fix

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:274769]: thank you!

**Pull request merged**